packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -1048,13 +1048,17 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin {
   /// Whether the [Scaffold.drawer] is opened.
   ///
   /// See also:
-  ///   * https://docs.flutter.io/flutter/material/ScaffoldState/openDrawer.html
+  ///
+  ///   * [ScaffoldState.openDrawer], which opens the [Scaffold.drawer] of a
```

Comment:
The `[ScaffoldState.openDrawer]` will automatically generate a link to https://docs.flutter.io/flutter/material/ScaffoldState/openDrawer.html, so you can remove the explicit link here and below.

Reply:
Oh, ok cool!